src/main/java/seedu/address/logic/parser/AddCommandParsers/AddMentorCommandParser.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.parser.AddCommandParsers;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ORGANISATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SPECIALISATION;
```

Comment:
Is specialization subject name? If so, should either put subject name or change the subjectName enum to specialization, both options are fine, just that I think it should be standardized.

Reply:
Yea you're right. I'll standardise it in the next pull request.